docs/unit-testing.md

```diff
@@ -0,0 +1,46 @@
+Jetpack includes several [unit tests](https://github.com/Automattic/jetpack/tree/master/tests) that you can run in your
+ local environment before to submit a new Pull Request.
+
+## Prerequisites
+
+To get started, you can follow the instructions [here](https://phpunit.de/getting-started.html) to install PHPUnit on 
+your machine. Once you've done so, you can get the WordPress testing codebase like so:
+
+You'll also need to install a mySQL server, to run the tests.
+
+# Option 1: Everything inside Jetpack folder
```

Comment:
Maybe some comments about what the difference between these two methods is? Will they both run the same tests, it's just a matter of where you're running them from?